### PR TITLE
Kyma release 1.12

### PR DESCRIPTION
**Pull request by dekiel**
Set versions for release 1.12

**Related issue(s)**
kyma-project/kyma#8252

### CODE REVIEW

File: installation/resources/installer.yaml [hunk 1]
```diff
@@ -201,7 +201,7 @@ spec:
             readOnly: true
       containers:
       - name: kyma-installer-container
-        image: eu.gcr.io/kyma-project/develop/installer:408cb6a6
+        image: eu.gcr.io/kyma-project/develop/installer:1.12
```
Comment (strekm) [inline installation/resources/installer.yaml:204]:
```suggestion
        image: eu.gcr.io/kyma-project/kyma-installer:1.12
```

**Comment by marcobebway** [comment]
Noticed this:

https://github.com/kyma-project/kyma/blob/5b2d311bc93a7e34b9e48bba7ee03950deeae3e3/resources/core/values.yaml#L49

**Pull request merged**